### DISCUSSION TITLE
notes/brittany-joiner/

### DISCUSSION THREAD

**Discussion by giscus** [dendronhq/cascadia-js-2022#41] [Announcements]
# notes/brittany-joiner/

Notes about the cascadiajs 2022 conference

https://cascadiajs.dendron.so/notes/brittany-joiner/

**Comment by vim-usds** [comment]
Slides here - https://www.canva.com/design/DAFD-lzA61U/v517F9SjB4mJz2laYc8U-Q/view#1

**Comment by vim-usds** [comment]
Slides here - https://slides.com/trivikram/cascadiajs-publishing/
